app/browser/api/ledger.js

```diff
@@ -1450,8 +1458,12 @@ const roundtrip = (params, options, callback) => {
     underscore.omit(params, ['headers', 'payload', 'timeout']))
 
 // TBD: let the user configure this via preferences [MTR]
-  if (parts.hostname === 'ledger.brave.com' && params.useProxy) {
-    parts.hostname = 'ledger-proxy.privateinternetaccess.com'
+  if (params.useProxy) {
+    if (parts.hostname === 'ledger.brave.com') {
```

Comment:
can we pull these up into a constant? could be a nice opportunity to name them in a more readable way.  For example, why are there two possible values? (ledger.brave.com and ledger.mercury.basicattentiontoken.org) The name of the variable may represent the usecase

Reply:
@mrose17 will not more about it, because he added them